### DISCUSSION TITLE
Add the possibility to resign without paid apple account

### DISCUSSION THREAD

**Discussion by danielpaulus** [danielpaulus/go-ios#180] [Ideas]
I think it's possible to use codesign to ad hoc codesign apps without the need of a paid account.
I'd really like to add that to the library so you can finally just install apps without the complicated setup.
 Does anyone know how?
See here: https://github.com/danielpaulus/wda-signer I am using a github action on a mac but it needs the paid mobileprovision and cert from a 99$/year account sadly.

**Comment by nanoscopic** [comment]
Free developer accounts work, you just have to re-sign weekly and re-install. It is annoying of course, but it does work.

As for signing more permanently, no?

All signing, either for a week or for a longer period are signed using Apple private keys using Apple services.

The best you can do is automate their endpoints ( which are mostly undocumented and build into Xcode )

**Reply by danielpaulus** [reply]
do you have some documentation around this or can you share more details on how it works exactly?
I'd like to integrate both ways of codesigning to go-ios.
